redbot/cogs/cleanup/cleanup.py

```diff
@@ -517,7 +512,6 @@ def check(m):
         await mass_purge(to_delete, channel)
 
     @cleanup.command(name="self")
-    @check_self_permissions()
```

Comment:
I dont think this was intentional, this is the reason that `    @checks.mod_or_permissions(manage_messages=True)` isnt added to the group like you done in this PR. I dont know enough about this check to comment on why just something i saw when skimming through it.

Reply:
Looks to me like a merge that went south. The checks in cleanup cog were touched somewhat recently and this seems to be accidentally reverting it.

Reply:
'Twas a revert to remove the diff from the PR from @Predeactor 
oh boi guise i'm unsure how to fix without closing and reopening with a with fresh V3/develop base 😟

Reply:
Let me know if that's best course, and I'll dip this PR, burn the branch, base from current V3/develop, write in the changes, and open new PR.

Sorry! ha